src/main/java/no/digipost/api/client/representations/Document.java

```diff
@@ -221,6 +221,10 @@ public boolean isOpened() {
         return opened != null && opened;
     }
 
+    public DataTypeHolder getDataType() {
+        return dataType;
+    }
```

Comment:
Hva er DataTypeHolder? Burde den eksponeres? Burde ikke den bare returnere `Optional<DataType>` i stedet?

Reply:
🤷‍♂️

Reply:
Jo, det virker vel fornuftig!

Reply:
Vi tar jo `DataType` inn som parameter. Og vi gjør det sånn kun for å få det til å passe med xsd. så 👍

Reply:
🤘